mava/wrappers/jumanji.py

```diff
@@ -135,7 +135,7 @@ def __init__(self, env: RobotWarehouse, add_global_state: bool = False):
     def modify_timestep(self, timestep: TimeStep) -> TimeStep[Observation]:
         """Modify the timestep for the Robotic Warehouse environment."""
         observation = Observation(
```

Comment:
For sanity check if you can run trace check to verify there is no double tracing because of types

Reply:
Checked and it seems fine :+1: